src/spikeinterface/core/sparsity.py

```diff
@@ -353,41 +357,35 @@ def from_snr(cls, templates_or_sorting_analyzer, threshold, noise_levels=None, p
         return cls(mask, unit_ids, channel_ids)
 
     @classmethod
-    def from_ptp(cls, templates_or_sorting_analyzer, threshold, noise_levels=None):
+    def from_ptp(cls, templates_or_sorting_analyzer, threshold):
```

Comment:
Is it worth adding a warning here that this method of computation was changed in this version, and to use `snr` with `peak_to_peak` to replicate the old method?

Reply:
Actually, I doupt anyone except @yger used it. I wouldn't overload with an additional warning :)

Reply:
Okay makes sense, anyone who is particularly interested can find it on the version release notes anyways

Reply:
Yes but here it is a strong change for a patch release no ?

Reply:
then let's release 0.102.0 ;)

I think Pierre was the sole user of this :P